hazelcast-jet-core/src/main/java/com/hazelcast/jet/Jet.java

```diff
@@ -130,6 +134,9 @@ static void configureJetService(JetConfig jetConfig) {
                         .setStatisticsEnabled(false)
                         .setMergePolicyConfig(
                                 new MergePolicyConfig().setPolicy(IgnoreMergingEntryMapMergePolicy.class.getName()))
+                )
+                .addMapConfig(new MapConfig(JOB_RESULTS_MAP_NAME)
```

Comment:
will this config take precedence over the wildcard?

Reply:
Exact match always takes precedence.